pkg/errors/errorUtils.go

```diff
@@ -0,0 +1,19 @@
+package errors
+
+import "fmt"
+
+func NewError(err error) error {
+	return err
+}
```

Comment:
It doesn't look like this function is being used; can it be removed? 🤔

Reply:
I added this in case we ever needed it in the future. Ultimately if people like this I'd like to port these methods to the go commons library so we can just adopt this standard across all the projects.